meilisearch/index.py

```diff
@@ -1881,24 +1913,68 @@ def get_embedders(self) -> Embedders | None:
         if not response:
             return None
 
-        embedders: dict[str, OpenAiEmbedder | HuggingFaceEmbedder | UserProvidedEmbedder] = {}
+        embedders: dict[
```

Comment:
If you defined this in the embedders file (line 203) can't you just use that definition here?